cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -1805,6 +1805,9 @@ def poke_network_unavailable():
 
     for node in nodes:
         node_name = node['metadata']['name']
+        poked_flag = 'kubernetes-master.gcp.network-poked.{}'.format(node_name)
+        if is_flag_set(poked_flag):
+            continue
```

Comment:
Can we safely assume the poke only needs to happen once? What if the node restarts? I would guess that we need to poke it again.